src/services/formatDate.ts

```diff
@@ -0,0 +1,5 @@
+import { DateTime } from "luxon";
+
+export const formatDate = (date: string): string => {
```

Comment:
We don't use this same format everywhere. You (rightly) brought up that that's something we might want to do. Until then, can you to specify this function name a bit more?